daemon/entrypoint.sh

```diff
@@ -292,7 +297,7 @@ function osd_directory {
     if [ ! -e /var/lib/ceph/osd/${CLUSTER}-${OSD_ID}/keyring ]; then
       chown ceph. /var/lib/ceph/osd/${CLUSTER}-${OSD_ID}
       # Create OSD key and file structure
-      ceph-osd ${CEPH_OPTS} -i $OSD_ID --mkfs --mkkey --mkjournal --osd-journal ${OSD_J} --setuser ceph --setgroup ceph
+      ceph-osd ${CEPH_OPTS} -i $OSD_ID --mkfs --mkkey --mkjournal --osd-journal ${OSD_J} --setuser ceph --setgroup disk
```

Comment:
I made this change in order to for block device journal creation to work. After much head pounding, and scouring, I found Issue https://github.com/ceph/ceph-docker/issues/171 and PR https://github.com/ceph/ceph-docker/pull/233. Changing group to "disk" allowed the ceph-disk operations to complete successfully.